src/backend/columnar/sql/citus_columnar--11.2-1.sql

```diff
@@ -0,0 +1 @@
+-- fake sql file 'Y'
```

Comment:
I do not think that we need this file at all.

Reply:
This file is for the scenario where user is working with an older citus version that predates citus_columnar and they want to upgrade to 11.1-1 or beyond. Pasting comment from Yanwen's PR https://github.com/citusdata/citus/pull/5911: 

```
Existing customer with older citus version and want upgrade to latest:

running "alter extension citus update to version Z" or "alter extension citus update":
In processUtility hook, before executing above commands, install citus_columnar to version 11.1-0 (version Y), to satisfies the requries = 'citus_columnar' in citus secondary control file citus--11.1-1.control
Ordinary update citus--11.0-2--11.1-1.sql : drop the columnar related objects from citus extension
In processUtility hook after running above commands, run citus_columnar--11.1-0--11.1-1.sql: to add previous columnar objects to citus_columnar extension
```